bluebutton-data-pipeline-fhir-load/src/main/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformer.java

```diff
@@ -985,4 +1084,100 @@ private static void upsert(Bundle bundle, Resource resource, String resourceUrl)
 
 		bundle.addEntry().setResource(resource).getRequest().setMethod(HTTPVerb.PUT).setUrl(resourceUrl);
 	}
+
+	/**
+	 * @param subPlan
+	 *            the {@link Coverage#getSubPlan()} value to match
+	 * @param subscriberPatientId
+	 *            the {@link Patient#getId()} for the
+	 *            {@link Coverage#getSubscriber()} value to match
+	 * @return a {@link Reference} to the {@link Coverage} resource where
+	 *         {@link Coverage#getPlan()} matches {@link #COVERAGE_PLAN} and the
+	 *         other parameters specified also match
+	 */
+	private static Reference referenceCoverage(String subscriberPatientId, String subPlan) {
+		return new Reference(String.format("Coverage?subscriber=%s&plan=%s&subplan=%s", subscriberPatientId,
+				COVERAGE_PLAN, subPlan));
+	}
+
+	/**
+	 * @param patientId
+	 *            the {@link Patient#getId()} value to match
+	 * @return a {@link Reference} to the {@link Patient} resource that matches
+	 *         the specified parameters
+	 */
+	private static Reference referencePatient(String patientId) {
+		return new Reference(String.format("Patient/%s", patientId));
+	}
+
+	/**
+	 * @param practitionerNpi
+	 *            the {@link Practitioner#getIdentifier()} value to match (where
+	 *            {@link Identifier#getSystem()} is
+	 *            {@value #CODING_SYSTEM_NPI_US})
+	 * @return a {@link Reference} to the {@link Practitioner} resource that
+	 *         matches the specified parameters
+	 */
+	static Reference referencePractitioner(String practitionerNpi) {
+		return new Reference(String.format("Practitioner/%s|%s", CODING_SYSTEM_NPI_US, practitionerNpi));
```

Comment:
Should this be:
`return new Reference(String.format("Practitioner?identifier=%s|%s", CODING_SYSTEM_NPI_US, practitionerNpi));`